README.md

```diff
@@ -220,15 +229,76 @@ manually you can do so by opening a sql shell.
       --certs-dir=${HOME}/.cockroachdb/certs/clients/root \
       --host localhost
 
-#### 4a. Setup cms database:
+#### 4a. Setup PostgresSQL databasee
+
+Install PostgresSQL using the instructions found in the [PostgresSQL
+Documentation](https://www.postgresql.org/download/macosx/).
+
+Run the following commands to create the PostgresSQL certificates required for
+running PostgresSQL with Politeia.
+
+    cd $GOPATH/src/github.com/decred/politeia
+    ./scripts/postgres/postgrescerts.sh
```

Comment:
Running this script created the root cert in the directory I was located in, not in the directories listed below.